runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -46,6 +46,9 @@
 
     static final long DEFAULT_BATCH_FOR_FAST_LOADER = 5;
     static final int DEFAULT_TIMEOUT_MINUTES_FAST_LOADING = 30;
+    @Getter
+    @Setter
+    static int defaultRetryMilliseconds = 5000;
```

Comment:
Upper case, sure.  Final, no, not without also chasing down many/all instances of tests creating their own CorfuRuntime objects and having all of instances configure non-default retry times.   A mutable class static var "hammer" is the right tool here, I think, to keep test times from creeping upward yet again.